src/Controllers/HelloWorldController.php

```diff
@@ -1,30 +1,23 @@
 <?php
 
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
-
 namespace App\Controllers;
 
-use Psr\Container\ContainerInterface;
 use Slim\Http\Request;
 use Slim\Http\Response;
-use Slim\Views\Twig as View;
+use Slim\Views\Twig;
 
 class HelloWorldController {
 
-    
-    protected $views;
+    private $twig;
+
+    public function __construct(Twig $twig) {
 
-    public function __construct(Views $views) {
+        $this->twig = $twig;
         
-        return $this->View = $views;
     }
 
-    /*public function __invoke(Request $request, Response $response, $args = []) {
-        return $views->render($response, 'helloworld.twig');
-    }*/
+    public function __invoke(Request $request, Response $response, $args = []) {
+        return $this->$twig->render($response, 'helloworld.twig');
```

Comment:
Cuando se llama a una propiedad se hace directamente sin poner el `$` de la propiedad, por lo que aquí simplemente bastaría con `$this->twig`, que es justamente como lo tenemos en el constructor